docs/modules/servers/pages/distributed/configure/collecting-events.adoc

```diff
@@ -60,4 +60,10 @@ A sample message looks like:
 	"sequence": "0",
 	"recurrence-id": null
 }
-....
\ No newline at end of file
+....
+
+ICALToHeader is represented by field `X-MEETING-UID` in the Header.
```

Comment:
The following pipeline positions the `X-MEETING-UID` in the Header in order for mail user agent to correlate events with this mail.

The sample look like:

...